io.github.sil_car.SqueezeVid.yaml

```diff
@@ -0,0 +1,126 @@
+# https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest
+app-id: io.github.sil_car.SqueezeVid
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: squeeze-vid
+
+finish-args:
+  - --filesystem=home
+
+cleanup-commands:
```

Comment:
no. you should se cleanup.

Also you should NOT delete `/lib/debug`

Reply:
what's the syntax for removing files/folders in `cleanup`? I didn't find that in the docs, which is why I used `cleanup-commands`.

Reply:
It's all documented there https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

or `man flatpak-manifest` for the manpage version.